test/packages/good_input/manifest.yml

```diff
@@ -58,6 +58,18 @@ policy_templates:
         required: true
         show_user: false
         default: "variables"
+      - name: sql_response_caching
+        type: select
+        title: Response caching
+        description: "Optional response caching time"
+        multi: false
+        required: true
+        show_user: false
+        options:
+          - value: "0"
+            text: None
+          - value: "1"
+            text: "1 day"
```

Comment:
Bit of a contrived example, a `select` would probably have been way more suitable for the `driver` or `sql_response_format` variables. Happy to change it if you want.

Reply:
That example looks good. With that we have a test also to check this field type for input packages :+1: